tna-eventbrite-api.php

```diff
@@ -17,7 +17,6 @@
 
 // add_action
 add_action('wp_enqueue_scripts', 'tna_ebapi_css');

```

Comment:
Undefined functions e.g. add_action, add_shortcode
This can be easily fixed by adding the if statement around all the actions